activity/inference/framework/tf/run.go

```diff
@@ -76,6 +75,7 @@ func (i *TensorflowModel) Run(model *models.Model) (out map[string]interface{},
 		default:
 
 			log.RootLogger().Info("Type not a Slice, Array, Map, or Pointer/Tensor, but still trying to make a tf.Tensor.")
+			fmt.Println(model.Inputs[inputName])
```

Comment:
remove println. replace with a proper log statement

Reply:
I have made the change and pushed it to here